### PR TITLE
Web documentation restructured

### PR DESCRIPTION
**Pull request by sibo86**
Web documentation has gotten an update - wording and fiddles. Guides have been added on each of the "tutorial" pages. A glossary is added under various.

### CODE REVIEW

**Review by marksdk — CHANGES_REQUESTED** [review]
Hey @sibo86,

I've commented here with a bunch of suggestions to changes. It looks like a lot, but it's small changes accumulated to a rather large review.

Let me know inline if I need to elaborate on a suggestion or comment.

Lastly, if you're using Visual Studio Code, there's a "Markdown linter" extension that will help the linting step go green as it fixes the issues it finds here when pushing code.

File: collections/_web/v3/guides/dynamic-updates.md [hunk 1]
```diff
@@ -80,12 +81,20 @@ bookings.onUpdate = (rooms) => {
...
 The callback is provided with a list of rooms and their statuses. The rooms are then divided into two arrays based on their status (`booked` or `notBooked`).
 
 Then the `DisplayRule`s are updated for each location in the two arrays. Doing this, we can color the room either green (for `notBooked`) or red (indicating the `booked` status).
 
----
+<br>
+
+******
+
+![guides](/assets/various/Guides.png "guides")
```
[the remark this answers is not in the document]
Comment (sibo86) [reply collections/_web/v3/guides/dynamic-updates.md:92]: I know that this is a hack hack as the Guides/Tutorial headlines wasn't easily accessible for using in this way - so it is made deliberately as I don't expect the page to "live" as is for a long period of time.

File: collections/_web/v3/guides/dynamic-updates.md [hunk 2]
```diff
@@ -80,12 +81,20 @@ bookings.onUpdate = (rooms) => {
...
-Now we register the `onUpdate` callback so we can update the map.
-
 The callback is provided with a list of rooms and their statuses. The rooms are then divided into two arrays based on their status (`booked` or `notBooked`).
 
 Then the `DisplayRule`s are updated for each location in the two arrays. Doing this, we can color the room either green (for `notBooked`) or red (indicating the `booked` status).
 
----
+<br>
+
+******
```
[the remark this answers is not in the document]
Comment (sibo86) [reply collections/_web/v3/guides/dynamic-updates.md:90]: The <br> tag is set to create some whitespace above the "hr" (which *** also forms). I cannot find any other way to make it so

File: collections/_various/glossary.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+See also Rooms.
+
+### Room
+
+A Room is a Location with polygon data.
+A Room might have one or more Doors connecting it with another or multiple other Rooms.
+
+### Area
+
+An Area is a geographic area covering a part of a Room’s polygon. It is not a Location, but a TK.
```
[the remark this answers is not in the document]
Comment (sibo86) [reply collections/_various/glossary.md:61]: Fine by me - must admit I more or less just copied what was in the document on Drive:)

File: collections/_various/glossary.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+### Building
+
+A Building covers an area within a Venue. A Building consists of one or more Floors.
+
+### Connector
+
+A Connector is the point at which a person can get from one Floor to another in a Building.
+The Connectors can be an Elevator, Stairs or Escalator.
+
+### Location / Place / POI
```
[the remark this answers is not in the document]
Comment (marksdk) [reply collections/_various/glossary.md:46]: @sibo86 This is the last outstanding issue. If there's a task to improve it later, we can keep it for now, but I think it needs a lift at some point regardless, and it's one of the things our SEs spend most time explaining to CMS users.

**Review by marksdk — APPROVED** [review]

**Pull request merged**